cmd/flarectl/flarectl.go

```diff
@@ -280,7 +279,7 @@ func zoneRecords(c *cli.Context) {
 		case "MX":
 			r.Content = fmt.Sprintf("%d %s", r.Priority, r.Content)
 		case "SRV":
-			dp := reflect.ValueOf(r.Data).Interface().(map[string]interface{})
+			dp := r.Data.(map[string]interface{})
```

Comment:
I just maintained the previous semantic, but yes, better error handling than just a panic would be good.